cmd/agent/subcommands/run/command.go

```diff
@@ -57,11 +53,15 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/telemetry"
 	"github.com/DataDog/datadog-agent/pkg/util"
 	"github.com/DataDog/datadog-agent/pkg/util/cloudproviders"
+	"github.com/DataDog/datadog-agent/pkg/util/constants"
 	"github.com/DataDog/datadog-agent/pkg/util/flavor"
 	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
 	"github.com/DataDog/datadog-agent/pkg/util/hostname"
 	pkglog "github.com/DataDog/datadog-agent/pkg/util/log"
 	"github.com/DataDog/datadog-agent/pkg/version"
+	"github.com/spf13/cobra"
```

Comment:
NIT: it seems like these imports are getting moved from the top section to the bottom here. Can you move them back (here and in other files)? (VSCode + the go plugin *should* respect this)

We generally keep `DataDog` level imports in their own group.

Reply:
done ✅